stable/entropy/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: entropy
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.2
```

Comment:
we can drop this to `0.1.0` to start with.

Reply:
oh right. that's a mistake. will fix it.